src/components/Notifications.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, useEffect } from 'react';
+
+interface NotificationsProps {
+  onClose: () => void,
+  errorMessage: string,
+}
+
+export const Notifications: FC<NotificationsProps> = (props) => {
+  const { onClose, errorMessage } = props;
+
+  useEffect(() => {
+    setTimeout(() => {
+      onClose();
+    }, 3000);
+  }, []);
```

Comment:
The `useEffect` hook is set to run only once (empty dependency array). This means that the timeout will be set only when the component is first mounted and not whenever the `errorMessage` prop changes. This may not be the desired behavior if you want the notification to automatically close 3 seconds after each error message is received.